status/manager.go

```diff
@@ -19,6 +19,10 @@ type SubServerStatus struct {
 	// been started.
 	Running bool
 
+	// Status will be a non-empty string that details the current status of
+	// the sub-server.
+	Status string
```

Comment:
I've updated the comment :)! The reason I currently have it duplicate info for "Registered", "Running", "Stopped" and "Errored", is that I thought it probably will be even more confusing for the user that it will else be blank from time to time in the response for the `litcli status` cmd. If you think that I should remove it I will though :). Else I think a perhaps better solution would have been to remove the booleans for the `litrpc.SubServerStatus` struct and only keep this new status field instead, but I'm not a huge fan of removing them for backwards compatibility reasons..

Let me know what you'd prefer :)!

Reply:
So I'd say that since it is called "manual status" it should be clear to the user that this is something special - so im thinking that we could probably just overwrite it to an empty string if any other status gets set.